index.js

```diff
@@ -0,0 +1,21 @@
+import { DateTime } from 'luxon';
```

Comment:
- [ ] There seems to be something wrong with your `luxon` that prevents me from navigating and using the app. You can see from the video below I installed the packages with `npm install` yet the luxon error still persists in the browser. Kindly look into that fix the errors

  [screen-capture (3).webm](https://user-images.githubusercontent.com/36542986/235436113-dc3a01c7-8f00-4218-9a21-3ea6b5a9082b.webm)